src/components/Filter/components/FilterSectionHeader/__tests__/filterSectionHeader.test.js

```diff
@@ -54,26 +54,26 @@ describe('FilterSectionHeader', () => {
     });
 
     it('shows the clear button when filters have been applied', () => {
-        const { getByText, container } = render(
+        const { getAllByText, container } = render(
             <FilterSectionHeader
                 {...FILTER_SECTION_PROPS}
                 value={['a', 'b', 'c']}
             />
         );
 
         expect(container.querySelector('svg[aria-label="clear"]')).toBeInTheDocument();
-        getByText(FILTER_SECTION_PROPS.filter.label);
+        getAllByText(FILTER_SECTION_PROPS.filter.label);
```

Comment:
Since you truly expect there to be multiple labels, should add a length check here (to make sure we don't have too many/few labels in the future).